src/ui/component/userEmailNew/view.jsx

```diff
@@ -39,12 +45,23 @@ function UserEmailNew(props: Props) {
           className="form-field--short"
           placeholder={__('hotstuff_96@hotmail.com')}
           type="email"
-          id="sign_up_email"
+          name="sign_up_email"
           label={__('Email')}
           value={newEmail}
           error={errorMessage}
           onChange={e => setEmail(e.target.value)}
         />
+        <FormField
+          type="checkbox"
+          name="sync_checkbox"
+          label={__('Sync your balance between devices')}
+          helper={
+            balance > 0 ? __('This is only available for empty wallets') : __('Maybe some more text about something')
```

Comment:
Copy check

Reply:
why is it only available for empty wallets?

Reply:
That is a current restriction of the sync API, though I don't think it will be like that for long.